python_examples/intervention_example/social_distancing_simulation_flow.py

```diff
@@ -30,104 +30,96 @@
-
-        if to_modify_parameter == 'distancing_enhanced_prob':
-            num_age_group = len(pe.Parameters.instance().intervention_params[
-                'social_distancing'][to_modify_parameter])
-            pe.Parameters.instance().intervention_params['social_distancing'][
-                to_modify_parameter] = [parameter_value] * num_age_group
-        else:
-            pe.Parameters.instance().intervention_params['social_distancing'][
-                'distancing_enhanced_prob'] = [0] * num_age_group
-            pe.Parameters.instance().intervention_params['social_distancing'][
-                to_modify_parameter] = parameter_value
-
-        print('Set {} to: {}'.format(to_modify_parameter,
-                                     pe.Parameters.instance(
-                                     ).intervention_params[
-                                      'social_distancing'][
-                                      to_modify_parameter]))
-
-        # Method to set the seed at the start of the simulation,
-        # for reproducibility
-        pe.routine.Simulation.set_random_seed(seed=30)
-
-        # Create population
-        population = pe.routine.FilePopulationFactory.make_pop(
-            file_loc, random_seed=42)
-
-        # Configure population with input data
-        pe.routine.ToyPopulationFactory.add_places(population, 1)
-
-        # file_params give details for where output should be written to.
-        file_params = {"output_file": name_output_file,
-                       "output_dir": os.path.join(
-                        os.path.dirname(__file__), "intervention_outputs"),
-                       "spatial_output": True,
-                       "age_stratified": True}
-
-        # Create a simulation object, configure it with the parameters given,
-        # then run the simulation.
-        sim = pe.routine.Simulation()
-        sim.configure(
-            population,
-            [pe.sweep.InitialInfectedSweep(), pe.sweep.InitialisePlaceSweep()],
-            [
-                pe.sweep.InterventionSweep(),
-                pe.sweep.UpdatePlaceSweep(),
-                pe.sweep.HouseholdSweep(),
-                pe.sweep.PlaceSweep(),
-                pe.sweep.SpatialSweep(),
-                pe.sweep.QueueSweep(),
-                pe.sweep.HostProgressionSweep(),
-            ],
-            sim_params,
-            file_params,
-        )
-        sim.run_sweeps()
-
-        # Need to close the writer object at the end of each simulation.
-        del sim.writer
-        del sim
+to_modify_parameter = 'distancing_enhanced_prob'
+parameter_values = [[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+                    [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1.0, 1.0, 1.0, 1.0,
+                     1.0],
+                    [1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0,
+                     1.0, 1.0, 1.0, 1.0, 1.0, 1.0]]
+labels = ['no intervention', 'SD elderly', 'SD']
```

Comment:
General point - we should be consistent on capitalisation on legends etc. I'm not worried about the demonstration plots in this repo, but keep an eye on this for anything going in the paper!